data/groups.json

```diff
@@ -157,7 +154,7 @@
     "nobonus": [],
     "tooltip": "https://liskitaliangroup.github.io",
     "color": "bg-green text-light",
-    "data": ["dakk", "liskit", "anamix", "corsaro", "splatters", "redsn0w", "gregorst", "ondin", "vekexasia", "fulig", "hirish"]
```

Comment:
Why removed `fulig` ?

Reply:
because he's not listed as member here: https://liskitalian.group/

but I realize, he's running the group. :man_facepalming: